addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -215,6 +215,7 @@ export class ComposerTextInput extends Component {
         }
         this._textareaLastInputValue = this._textareaRef.el.value;
         this._updateHeight();
+        this.composerView.composer.messageComposition.removeSuggestions();
```

Comment:
Does it fix a bug?

Could be another PR in v14 or v15 if that's the case, and this line would simply be a move from `Composer/removeSuggestions` to `MessageComposition/removeSuggestions` (which makes sense in the context of this PR).

Reply:
It fix a bug introduced by the PR :)

Reply:
You are right, in fact, it's doesn't look useful at all. I don't remember exactly why I did it in the first place, but I guess it doesn't matter anymore :)